util/types/json/json_test.go

```diff
@@ -27,19 +28,28 @@ func TestT(t *testing.T) {
 	TestingT(t)
 }
 
+func mustParseFromString(s string) JSON {
+	j, err := ParseFromString(s)
+	if err != nil {
+		msg := fmt.Sprintf("ParseFromString(%s) fail", s)
+		panic(msg)
+	}
+	return j
+}
+
+func (s *testJSONSuite) TestParseFromString(c *C) {
+	jstr1 := `{"a": [1, "2", {"aa": "bb"}, 4, null], "b": true, "c": null}`
+	jstr2 := mustParseFromString(jstr1).String()
+	c.Assert(jstr2, Equals, `{"a":[1,"2",{"aa":"bb"},4,null],"b":true,"c":null}`)
+}
+
 func (s *testJSONSuite) TestJSONSerde(c *C) {
```

Comment:
`Serde` is not a commonly used short name.
Please use full words.
And `JSON` can be removed as the test suite is already named JSON.